client/components/async-gridicons/index.jsx

```diff
@@ -13,8 +13,10 @@ import FallbackIcon from './fallback';
 const loadedIcons = new Map();
 
 function loadIcon( icon ) {
-	return import( /* webpackChunkName: "gridicons", webpackInclude: /\.js$/, webpackExclude: /dist\/(index\.js|example\.js)$/, webpackMode: "lazy-once" */
-	`gridicons/dist/${ icon }` ).then(
+	return import(
+		/* webpackChunkName: "gridicons", webpackInclude: /\.js$/, webpackExclude: /dist\/(index\.js|example\.js)$/, webpackMode: "lazy-once" */
```

Comment:
The magic comments can be broken up which would make these more readable:

```
// Multiple possible targets
import(
  /* webpackInclude: /\.json$/ */
  /* webpackExclude: /\.noimport\.json$/ */
  /* webpackChunkName: "my-chunk-name" */
  /* webpackMode: "lazy" */
  /* webpackPrefetch: true */ 
  /* webpackPreload: true */
  `./locale/${language}`
);
```

https://webpack.js.org/api/module-methods/#magic-comments

Reply:
Unfortunately, the Prettier formatting of these broken up comments is not so nice as expected. I might file an issue.